src/test/java/com/varchenko/FibonacciSequenceTest.java

```diff
@@ -0,0 +1,29 @@
+package com.varchenko;
+
+import static org.junit.jupiter.api.Assertions.*;
+import org.junit.jupiter.api.Test;
+import static com.varchenko.FibonacciSequence.fib;
+
+public class FibonacciSequenceTest {
+    FibonacciSequence fibonacciSequence = new FibonacciSequence() ;
+        @Test
+        void Fibonacci_numbersFrom0to10() {
+            for (int index = 0; index < 10; index++) {
```

Comment:
You don't have to test several results in one test.
Tests are atomic. 
For one use case, we use 1 test.

By the way, what the difference between a test with value 6 and a value 8?

Reply:
You had to add some edge case tests. 

For example, test what happens if requested 1, 0, -1, 5000.
Please add the following tests. 

Besides being a good practice, they will show you that your code is pretty undeveloped :)

Reply:
done